src/frontend/components/property-type/currency/show.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { PureComponent, ReactNode } from 'react'
+import { ValueGroup } from '@adminjs/design-system'
+
+import { EditPropertyProps } from '../base-property-props'
+import formatValue from './format-value'
+
+export default class Show extends PureComponent<EditPropertyProps> {
```

Comment:
does this have to be a class component for some reason? I also noticed that some other old components are class components but I'm not sure if there's a reason.

Reply:
As you mentioned I created this component based on another one but true should be here consistent and I changed to FC. 
Thanks for pointing that!